icu4c/source/i18n/unicode/measunit.h

```diff
@@ -439,6 +439,20 @@ class U_I18N_API MeasureUnit: public UObject {
      * @draft ICU 67
      */
     MeasureUnit product(const MeasureUnit& other, UErrorCode& status) const;
+
+    /**
+     * Extracts a simplified version of the unit (i.e. no repetition in the internal units). For example,
+     * "square-meter-per-meter" will be "meter".
+     *
+     * NOTE: Only works on SINGLE and COMPOUND units. If the current unit is a
+     * MIXED unit, an error will occur. For more information, see UMeasureUnitComplexity.
+     *
+     * @param status Set if this is a MIXED unit or if another error occurs.
+     * @return a simplified version of the current unit.
+     * @draft ICU 67
+     */
+    MeasureUnit simplify( UErrorCode& status) const;
```

Comment:
This is a new public API. As I understood it, the units were not simplified automaticaly, because it's needed for disambiguation. If we can find a nice example of it (of what is "lost"), this comment should probably include it. (I can't search right now.)

If we're adding a public API, I think it needs a couple of unit tests too.

Reply:
Do you mean you will add some tests? MeasureUnit tests live in measfmttest.cpp if I recall correctly.

Reply:
https://github.com/icu-units/icu/issues/1

Reply:
Using tickets LGTM. Marking as resolved. Thanks!